tools/pdf2txt.py

```diff
@@ -16,6 +16,15 @@
                 (".tag", "tag"))
 
 
+def float_or_none(x):
+    if x == "None":
```

Comment:
This can be more forgiving. 
Also, since it parses the string anyway, it could also be another string. "disabled" would make sense to me, but feel free to use the one that makes most sense to you. 

```python
if x.lower().strip() == "disabled":
```